bubuku/features/restart_if_dead.py

```diff
@@ -20,8 +20,7 @@ def check(self) -> Change:
             return None
         if self.broker.is_running_and_registered():
             return None
-        if self.zk.is_rolling_restart_in_progress():
-            return None
```

Comment:
Seems that this was written for a purpose (do not start again, as commands are independent)
I mean during test we can find it already, but it may make sense to fix it beforehands

Reply:
Hm, not sure how is this going to work then.  If the other broker doesn't signal start anymore, we will not try to start it either.  And will not try to start again in case first attempt failed (which was exactly the issue).

Reply:
So the thing is, that in the rolling restart there are 2 roles - leader and follower. 
For now leader is issuing commands, and follower is just running them. 

What is happening - leader issues Stop() command, follower stops, Leader issues Start() - follower starts. 
And in this case, if follower will stop as a reaction to `Stop` command, then it will immediately start as a part of `restart_if_dead` check. And it will be followed by termination from leader (or hangup of leader command execution).

Therefore I suggest to have expected state of follower - "stopped", and not to issue `Stop` command. Then follower will be able just to remove this expected state and follower will be moving towards proper state as part of usual lifecycle process.

Reply:
@a1exsh (forgot to tag)